src/formState.ts

```diff
@@ -464,14 +464,22 @@ function newObjectState<T, P = any>(
     get changedValue() {
       const result: any = {};
       getFields(this).forEach((f) => {
-        // If the called used useFormState.ifUndefined, the key may not look dirty even though we're new
-        if (f.dirty || this.isNewEntity) {
```

Comment:
This `|| this.isNewEntity` didn't exist when the TP contact page was originally built, so explains why it worked at the time.